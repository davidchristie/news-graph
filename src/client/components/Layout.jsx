import React, {PropTypes} from 'react'

import Footer from './Footer'
import Navbar from './Navbar'

export default class Layout extends React.Component {
  render () {
    const children = this.props.children
    return (
      <div>
        <Navbar />
        {children}
        <Footer />
      </div>
    )
  }
}

Layout.propTypes = {
  children: PropTypes.element
}
