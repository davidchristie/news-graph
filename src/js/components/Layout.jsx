import React, {PropTypes} from 'react'

import Header from './Header'
import Footer from './Footer'

export default class Layout extends React.Component {
  render() {
    const children = this.props.children
    return (
      <div>
        <Header />
        {children}
        <Footer />
      </div>
    )
  }
}

Layout.propTypes = {
  children: PropTypes.element
}
