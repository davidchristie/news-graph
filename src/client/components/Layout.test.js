/* eslint-env jest */

import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Layout from './Layout'

test('Layout renders correctly', () => {
  const wrapper = shallow(<Layout />)
  expect(toJson(wrapper)).toMatchSnapshot()
})
