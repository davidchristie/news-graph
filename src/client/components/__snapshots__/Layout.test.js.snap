exports[`test Layout renders correctly 1`] = `
<div>
  <Navbar />
  <Footer />
</div>
`;
